sandbox/registration-processor-mz.properties

```diff
@@ -446,6 +446,8 @@ registration.processor.policy.id=mpolicy-default-abis
 registration.processor.subscriber.id=mpartner-default-abis
 mosip.regproc.data.share.internal.domain.name=datashare-service
 mosip.regproc.data.share.protocol=http
+mosip.regproc.data.share.modalities.segments={'Finger': {'Left Thumb','Left LittleFinger','Left IndexFinger','Left MiddleFinger','Left RingFinger','Right Thumb','Right LittleFinger','Right IndexFinger','Right MiddleFinger','Right RingFinger'},'Iris':{'Left','Right'},'Face':{'Face'}}
```

Comment:
Can you rename this config to mosip.regproc.abis.handler.biometric-modalities-segments-mapping and the move it to the place where all the mosip.regproc.abis.handler. configs are there ? 

Do the same with next config also

Reply:
Incorporated review comments